{
  "comments": [
    {
      "key": {
        "uuid": "64455f17_c8f61e61",
        "filename": "modules/rtp_rtcp/source/rtp_sequence_number_map.cc",
        "patchSetId": 1
      },
      "lineNbr": 87,
      "author": {
        "id": 5019
      },
      "writtenOn": "2019-04-05T13:28:54Z",
      "side": 1,
      "message": "so packet_count \u003d\u003d std::numeric_limits\u003csize_t\u003e::max() is not valid, but\nstd::numeric_limits\u003csize_t\u003e::max() - 1 is valid.\nwhy?\n\nmay be you mean LE(\u003cuint16_t\u003e::max() instead?",
      "range": {
        "startLine": 87,
        "startChar": 50,
        "endLine": 87,
        "endChar": 56
      },
      "revId": "2190e875e30cd605eb947d85380a93dfe66edc36",
      "serverId": "58829da1-049c-39fb-b951-ebdcd0984767",
      "unresolved": false
    }
  ]
}